packages/core/components/FileDetails/index.tsx

```diff
@@ -86,15 +88,59 @@ export default function FileDetails(props: FileDetails) {
 
     // If the file has a thumbnail image specified, we want to display the specified thumbnail. Otherwise, we want
     // to display the file itself as the thumbnail if possible.
-    let thumbnailUriPath = "";
+    // If there is no thumbnail and the file cannot be displayed as the thumbnail- show a no image icon
+    let thumbnail = null;
     if (fileDetails?.thumbnail) {
-        thumbnailUriPath = fileDetails.thumbnail;
+        // thumbnail exists
+        thumbnail = (
+            <div
+                className={classNames(styles.fileThumbnailContainer, {
+                    [styles.thumbnailDefault]: windowState.state === WindowState.DEFAULT,
+                    [styles.thumbnailMaximized]: windowState.state === WindowState.MAXIMIZED,
+                })}
+            >
+                <FileThumbnail
+                    uri={`http://aics.corp.alleninstitute.org/labkey/fmsfiles/image${fileDetails.thumbnail}`}
+                />
+            </div>
+        );
     } else if (fileDetails) {
         const renderableImageFormats = [".jpg", ".jpeg", ".png", ".gif"];
         const isFileRenderableImage = renderableImageFormats.some((format) =>
             fileDetails?.name.toLowerCase().endsWith(format)
         );
-        if (isFileRenderableImage) thumbnailUriPath = fileDetails.path;
+        if (isFileRenderableImage) {
+            // render the image as the thumbnail
+            thumbnail = (
+                <div
+                    className={classNames(styles.fileThumbnailContainer, {
+                        [styles.thumbnailDefault]: windowState.state === WindowState.DEFAULT,
+                        [styles.thumbnailMaximized]: windowState.state === WindowState.MAXIMIZED,
+                    })}
+                >
+                    <FileThumbnail
+                        uri={`http://aics.corp.alleninstitute.org/labkey/fmsfiles/image${fileDetails.path}`}
+                    />
+                </div>
+            );
+        } else {
+            //show placeholder
+            thumbnail = (
+                <div>
+                    <SvgIcon
+                        height={100}
+                        pathData={NO_IMAGE_ICON_PATH_DATA}
+                        viewBox="0,0,20,20"
+                        width={100}
+                        className={classNames(styles.fileThumbnailContainer, styles.noThumbnail, {
+                            [styles.thumbnailDefault]: windowState.state === WindowState.DEFAULT,
+                            [styles.thumbnailMaximized]:
+                                windowState.state === WindowState.MAXIMIZED,
+                        })}
+                    />
+                </div>
+            );
+        }
     }
```

Comment:
@GabeMedrash Is this too verbose? I initially created another if else statement after the getting the thumbnailURI to create the thumbnail, but I thought this was cleaner since we don't have to save the thumbnailURI.

Reply:
I dunno about Gabe, but I find this much more readable.  Remember, the person (maybe future you) who comes back to work on this latter may be stressed, up too late and under time pressure-clarity and ease of understanding are powerful.

Reply:
Brian--not at all, I think the code communicates well. Here's one suggestion for a minor improvement (cuts out just a tiny bit of cyclomatic complexity): instead of initializing `thumbnail` to `null`, you could initialize it to the "no image" placeholder. 

Also, one more question for you: is the `div` that wraps the `svg` necessary? What's it doing?